src/index.html

```diff
@@ -9,6 +9,22 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="container">
+      <header class="header">
+        <img src="images/logo.png" alt="logo"
+        class="logo">
+        <nav class="nav">
+          <a href="#Apple" class="nav__link is-active">Apple</a>
```

Comment:
```suggestion
          <a 
            href="#Apple" 
            class="nav__link is-active"
          >
            Apple
         </a>
```
Fix formatting for similiar cases